### ISSUE TITLE
`config.nims` breaks `.nim.cfg`

### ISSUE THREAD

**Issue by arnetheduck** [status-im/nim-beacon-chain#757]
per https://github.com/status-im/nim-beacon-chain/blob/master/research/state_sim.nim.cfg, state_sim should be compiled without metrics, but is compiled with. this is because config.nims gets applied after the module-specific config, as of recently.

**Comment by Clyybber** [comment]
Fixed in devel: https://github.com/nim-lang/Nim/pull/13488. It will be backported.

**Comment by stefantalpalaru** [comment]
The parsing order was fixed in 1.2.0:

```text
stefan$ ./env.sh nim c research/state_sim.nim
Hint: used config file '/mnt/sda3/storage/CODE/status/nim-beacon-chain-clean/vendor/nimbus-build-system/vendor/Nim/config/nim.cfg' [Conf]
Hint: used config file '/mnt/sda3/storage/CODE/status/nim-beacon-chain-clean/vendor/nimbus-build-system/vendor/Nim/config/config.nims' [Conf]
Hint: used config file '/mnt/sda3/storage/CODE/status/nim-beacon-chain-clean/config.nims' [Conf]
Hint: used config file '/mnt/sda3/storage/CODE/status/nim-beacon-chain-clean/research/state_sim.nim.cfg' [Conf]
```
